### DISCUSSION TITLE
disable module in ltsp build

### DISCUSSION THREAD

**Discussion by wforumw** [ltsp/ltsp#421] [Q&A] [answered]
Hi,
We use all the same clients and would like to disable the wifi module. In the bios we cannot disable the adapter.
How can I disable the module in the LTSP image.
Vielen dank

**Comment by alkisg** [comment, marked as answer]
One way to do it is with something like this in ltsp.conf:

```shell
[clients]
POST_INIT_BLACKLIST_MODULE="rs rmmod iwlwifi; echo 'blacklist iwlwifi' > /etc/modprobe.d/local.conf"
```

Replace `iwlwifi` with the module that corresponds to your internal wifi adapter.

**Reply by wforumw** [reply]
Thanks it works
